src/main/java/entities/Week.java

```diff
@@ -0,0 +1,16 @@
+package entities;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Week{
+    private final ArrayList<Day> days;
+
+    public Week(ArrayList<Day> days){
+        this.days = days;
+    }
+
```

Comment:
add a default constructor that accomplishes the same thing as the existing constructor but with no parameters